apps/hash-ai-worker-ts/src/activities/shared/get-llm-response.ts

```diff
@@ -666,15 +689,170 @@ const getOpenAiResponse = async (
  * `model` provided in the parameters.
  */
 export const getLlmResponse = async <T extends LlmParams>(
-  params: T,
+  params: T & {
+    /**
+     * Required for tracking usage on a per-user basis.
+     *
+     * @todo: consider abstracting this in a wrapper method, or via
+     * generic params (via a `logUsage` method).
+     */
+    userAccountId: AccountId;
+    graphApiClient: GraphApi;
+    linkUsageRecordToEntities: {
+      linkEntityTypeId: [
+        typeof systemLinkEntityTypes.incurredIn.linkEntityTypeId,
+        typeof systemLinkEntityTypes.created.linkEntityTypeId,
+        typeof systemLinkEntityTypes.updated.linkEntityTypeId,
```

Comment:
We never pass these as far as I can see. Probably we can just take the `entityId`

Reply:
Yes, we can add this back once we have a need for using other links. Have simplified this param in https://github.com/hashintel/hash/pull/4379/commits/80b1df03da2aba99b964922add7e3f97bf2dfe79